packages/design-system/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, forwardRef, HTMLAttributes, ReactElement, Ref } from 'react';
+
+import styles from './Breadcrumbs.module.scss';
+import Link from '../Link';
+import Dropdown from '../Dropdown/Dropdown';
+import { ButtonTertiary } from '../Button';
+import { StackHorizontal } from '../Stack';
+
+type BreadCrumbItems = {
+	label: string;
+	href: string;
+	target?: '_blank' | '_self';
+}[];
+
+type BreadCrumbProps = Omit<HTMLAttributes<HTMLElement>, 'className'> & {
+	as?: 'a' | ReactElement;
+	items: BreadCrumbItems;
+};
+
+const Breadcrumbs = forwardRef(
+	({ as = 'a', items, ...rest }: BreadCrumbProps, ref: Ref<HTMLElement>) => {
+		const maxItemLength = 4;
+
+		const buildEntries = useCallback(() => {
+			if (items.length > maxItemLength) {
+				const origin = items[0];
+				const suffix = items.slice(-2);
+				const collapsed = items.slice(1, items.length - 2);
+
+				return (
+					<>
+						<li className={styles.entry}>
+							<Link href={origin.href} as={as} target={origin.target}>
```

Comment:
I'm more in favor to spread attrs here, if I want to pass an icon, or a data-attribute